tools/integration_tests/operations/copy_dir_test.go

```diff
@@ -220,3 +219,147 @@ func TestCopyDirectoryInNonEmptyDirectory(t *testing.T) {
 	destSrc := path.Join(destDir, SrcCopyDirectory)
 	checkIfCopiedDirectoryHasCorrectData(destSrc, t)
 }
+
+func checkIfCopiedEmptyDirectoryHasCorrectData(destSrc string, t *testing.T) {
```

Comment:
Should I rename [this](https://github.com/GoogleCloudPlatform/gcsfuse/blob/master/tools/integration_tests/operations/copy_dir_test.go#L68) to checkIfCopiedNonEmptyDirectoryHasCorrectData ?

Reply:
Empty directory means no data. CorrectData doesn't make sense for empty directory.

Instead we should rename this to - checkIfCopiedEmptyDirectoryHasNoData or similar.